src/client/pages/OfferNew/InsuranceSelector/Selector.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { useState, KeyboardEvent, useEffect } from 'react'
+import styled from '@emotion/styled'
+import { colorsV3 } from '@hedviginsurance/brand'
+import { MEDIUM_SCREEN_MEDIA_QUERY } from 'utils/mediaQueries'
+import { Card } from './Card'
+
+const CardListContainer = styled.div`
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-auto-rows: auto;
+  gap: 1rem;
+  background-color: ${colorsV3.gray100};
+  outline: none;
+
+  ${MEDIUM_SCREEN_MEDIA_QUERY} {
+    grid-template-columns: 1fr 1fr;
+  }
+`
+
+const CardWrapper = styled.div`
+  outline: none;
+  cursor: pointer;
+`
+
+interface Props {
```

Comment:
Should we be consistent with using `type`? :)

Reply:
Sure, I do agree that consistency is valuable!